.nix/fallback-config.nix

```diff
@@ -0,0 +1,79 @@
+# This file was generated from `meta.yml`, please do not edit manually.
+# Follow the instructions on https://github.com/coq-community/templates to regenerate.
+
+# *However*, you may copy paste it to `config.nix`
+# in the same subdirectory .nix and extend it as needed.
+# Note that this would completely deactivate this file.
+
+{
+  ## DO NOT CHANGE THE format
+  format = "1.0.0";
+  ## unless you made an automated or manual update
+  ## to another supported format.
+
+  ## The attribute to build, either from nixpkgs
+  ## of from the overlays located in `.nix/coq-overlays`
+  attribute = "lemma-overloading";
+
+  ## If you want to select a different attribute
+  ## to serve as a basis for nix-shell edit this
+  # shell-attribute = "lemma-overloading";
+
+  ## Maybe the shortname of the library is different from
+  ## the name of the nixpkgs attribute, if so, set it here:
+  pname = "lemma-overloading";
+
+  ## Lists the dependencies, phrased in terms of nix attributes.
+  ## No need to list Coq, it is already included.
+  ## These dependencies will systematically be added to the currently
+  ## known dependencies, if any more than Coq.
+  ## /!\ Remove this field as soon as the package is available on nixpkgs.
+  ## /!\ Manual overlays in `.nix/coq-overlays` should be preferred then.
+  buildInputs = [ "mathcomp-ssreflect" ];
```

Comment:
@Zimmi48 this is the culprit! This specific set of `buildInputs` is only added by `mkDeriv`, it should be added before...

Reply:
This line should really be used as a last resort anyway, please use an overlay instead :)
(and report the current bug on coq-nix-toolbox)

Reply:
> This line should really be used as a last resort anyway, please use an overlay instead :)

I was trying to use the toolbox through the templates infrastructure. This means generating everything from `meta.yml` instead of writing overlays manually.